src/main/scala/com/microsoft/hyperspace/index/IndexCollectionManager.scala

```diff
@@ -163,6 +162,14 @@ class IndexCollectionManager(
     }
   }
 
+  private[hyperspace] override def getIndexLogEntry(
```

Comment:
If you check https://github.com/microsoft/hyperspace/pull/272#discussion_r579565340, I think it's fine to expose this as public (this is accessed from source relation, meaning it may be helpful to expose publicly), as long as we also expose APIs to get the log versions.